drivers/Watchdog.h

```diff
@@ -15,26 +15,29 @@
  * limitations under the License.
  */
 
-#ifndef MBED_WATCHDOG_H
-#define MBED_WATCHDOG_H
+#ifndef MBED_SW_WATCHDOG_H
```

Comment:
You could probably remove `SW` from this name since this is just called watchdog.